[ 2024-05-01 10:14:16,880 ] 91 root - INFO - Starting all threads
[ 2024-05-01 10:14:16,880 ] 91 root - INFO - Starting all threads
[ 2024-05-01 10:14:16,881 ] 91 root - INFO - Meta: {'__module__': 'src.globals', '__doc__': '        Meta configs        ', 'keyword': '行政', 'test_mode': True, 'debug_mode': True, 'onlyday': True, 'max_num_retry': 5, 'default_num_threads': 3, 'display_number': 50, 'main_phone_number_dash': '080-1583-0891', 'uc': True, 'headless': False, 'timeout': 900, '__dict__': <attribute '__dict__' of 'Meta' objects>, '__weakref__': <attribute '__weakref__' of 'Meta' objects>}
[ 2024-05-01 10:14:16,881 ] 91 root - INFO - Meta: {'__module__': 'src.globals', '__doc__': '        Meta configs        ', 'keyword': '行政', 'test_mode': True, 'debug_mode': True, 'onlyday': True, 'max_num_retry': 5, 'default_num_threads': 3, 'display_number': 50, 'main_phone_number_dash': '080-1583-0891', 'uc': True, 'headless': False, 'timeout': 900, '__dict__': <attribute '__dict__' of 'Meta' objects>, '__weakref__': <attribute '__weakref__' of 'Meta' objects>}
[ 2024-05-01 10:14:16,882 ] 91 root - INFO - Number of dev accounts: 3
[ 2024-05-01 10:14:16,882 ] 91 root - INFO - Number of dev accounts: 3
[ 2024-05-01 10:14:16,882 ] 91 root - INFO - Test mode: True
[ 2024-05-01 10:14:16,882 ] 91 root - INFO - Test mode: True
[ 2024-05-01 10:14:16,888 ] 91 main_controller - INFO - Subprocess 0 started with trial number 0
[ 2024-05-01 10:14:16,889 ] 91 main_controller - ERROR - Account is a dictionary, convert to Account object: Username: luthien5921@gmail.com, Password: aichi@5921
[ 2024-05-01 10:14:18,895 ] 91 root - INFO - Thread 0 started
[ 2024-05-01 10:14:18,895 ] 91 root - INFO - Thread 0 started
[ 2024-05-01 10:14:18,900 ] 91 main_controller - INFO - Subprocess 1 started with trial number 0
[ 2024-05-01 10:14:18,901 ] 91 main_controller - ERROR - Account is a dictionary, convert to Account object: Username: luthien5921+shinsei1@gmail.com, Password: aichi@5921
[ 2024-05-01 10:14:19,869 ] 91 main_controller - INFO - Browser opened
[ 2024-05-01 10:14:20,907 ] 91 root - INFO - Thread 1 started
[ 2024-05-01 10:14:20,907 ] 91 root - INFO - Thread 1 started
[ 2024-05-01 10:14:20,922 ] 91 main_controller - INFO - Subprocess 2 started with trial number 0
[ 2024-05-01 10:14:20,923 ] 91 main_controller - ERROR - Account is a dictionary, convert to Account object: Username: luthien5921+shinsei2@gmail.com, Password: aichi@5921
[ 2024-05-01 10:14:22,635 ] 91 main_controller - INFO - Browser opened
[ 2024-05-01 10:14:22,927 ] 91 root - INFO - Thread 2 started
[ 2024-05-01 10:14:22,927 ] 91 root - INFO - Thread 2 started
[ 2024-05-01 10:14:28,508 ] 91 luthien5921 - INFO - Opened login page
[ 2024-05-01 10:14:28,510 ] 91 luthien5921+shinsei2 - INFO - Opened login page
[ 2024-05-01 10:14:32,782 ] 91 luthien5921+shinsei2 - INFO - Logging in
[ 2024-05-01 10:14:32,789 ] 91 luthien5921+shinsei2 - INFO - Current url: https://www.shinsei.e-aichi.jp/pref-aichi-police-u/offer/offerList_initDisplay
[ 2024-05-01 10:14:32,846 ] 91 luthien5921 - INFO - Logging in
[ 2024-05-01 10:14:32,852 ] 91 luthien5921 - INFO - Current url: https://www.shinsei.e-aichi.jp/pref-aichi-police-u/offer/offerList_initDisplay
[ 2024-05-01 10:14:33,101 ] 91 luthien5921+shinsei2 - INFO - Logged in finished
[ 2024-05-01 10:14:33,102 ] 91 main_controller - INFO - Logged in successfully
[ 2024-05-01 10:14:33,163 ] 91 luthien5921 - INFO - Logged in finished
[ 2024-05-01 10:14:33,164 ] 91 main_controller - INFO - Logged in successfully
[ 2024-05-01 10:14:33,480 ] 91 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_05_01\10_14_15\screenshot_luthien5921+shinsei2.png
[ 2024-05-01 10:14:33,481 ] 91 main_controller - INFO - Finding forms
[ 2024-05-01 10:14:33,492 ] 91 luthien5921+shinsei2 - INFO - On main page
[ 2024-05-01 10:14:33,494 ] 91 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_05_01\10_14_15\screenshot_luthien5921.png
[ 2024-05-01 10:14:33,495 ] 91 main_controller - INFO - Finding forms
[ 2024-05-01 10:14:33,503 ] 91 luthien5921 - INFO - On main page
[ 2024-05-01 10:14:35,329 ] 91 luthien5921+shinsei2 - INFO - Search submitted
[ 2024-05-01 10:14:35,358 ] 91 luthien5921+shinsei2 - INFO - Collected 2 forms
[ 2024-05-01 10:14:35,444 ] 91 luthien5921+shinsei2 - INFO - Number of filtered forms left is: 2 form(s)
[ 2024-05-01 10:14:35,445 ] 91 luthien5921+shinsei2 - INFO - 行政文書任意開示申出書 -  - 2015年08月19日 09時00分 - 2100年1月1日 00時00分 - 3895 - https://www.shinsei.e-aichi.jp/pref-aichi-police-u/offer/offerList_detailTop?tempSeq=3895 - False
[ 2024-05-01 10:14:35,445 ] 91 luthien5921+shinsei2 - INFO - 行政文書開示請求書 -  - 2015年08月19日 09時00分 - 2100年1月1日 00時00分 - 3896 - https://www.shinsei.e-aichi.jp/pref-aichi-police-u/offer/offerList_detailTop?tempSeq=3896 - False
[ 2024-05-01 10:14:35,446 ] 91 main_controller - INFO - Found 2 form(s)
[ 2024-05-01 10:14:35,447 ] 91 main_controller - INFO - Customers: [{'first_name': 'The Hieu', 'last_name': 'Truong', 'date_birth': '19990506', 'nation': 'ベトナム', 'country': 'ベトナム'}, {'first_name': 'Xuan Truong', 'last_name': 'Do', 'date_birth': '19921211', 'nation': 'ベトナム', 'country': 'ベトナム'}, {'first_name': 'Anh Ngoc', 'last_name': 'Chia', 'date_birth': '19961208', 'nation': 'ベトナム', 'country': 'ベトナム'}, {'first_name': 'Van Sang', 'last_name': 'Phan', 'date_birth': '19970228', 'nation': 'ベトナム', 'country': 'ベトナム'}, {'first_name': 'Hoai Phong', 'last_name': 'Vo', 'date_birth': '19981101', 'nation': 'ベトナム', 'country': 'ベトナム'}, {'first_name': 'Tuan Anh', 'last_name': 'Huynh', 'date_birth': '19950106', 'nation': 'ベトナム', 'country': 'ベトナム'}, {'first_name': 'Van Han', 'last_name': 'Nguyen', 'date_birth': '19910525', 'nation': 'ベトナム', 'country': 'ベトナム'}]
[ 2024-05-01 10:14:35,447 ] 91 main_controller - INFO - Customer info index 0: {'first_name': 'The Hieu', 'last_name': 'Truong', 'date_birth': '19990506', 'nation': 'ベトナム', 'country': 'ベトナム', 'phone_number': '080-1583-0891', 'gender': 'M'}
[ 2024-05-01 10:14:35,448 ] 91 main_controller - INFO - Customer index 0: Truong The Hieu, M, 19990506, 080-1583-0891, ベトナム
[ 2024-05-01 10:14:35,448 ] 91 luthien5921+shinsei2 - INFO - Accessing form, template sequence: 3896
[ 2024-05-01 10:14:35,449 ] 91 luthien5921+shinsei2 - INFO - Form URL: https://www.shinsei.e-aichi.jp/pref-aichi-police-u/offer/offerList_detailTop?tempSeq=3896
[ 2024-05-01 10:14:35,466 ] 91 luthien5921 - INFO - Search submitted
[ 2024-05-01 10:14:35,505 ] 91 luthien5921 - INFO - Collected 2 forms
[ 2024-05-01 10:14:35,594 ] 91 luthien5921 - INFO - Number of filtered forms left is: 2 form(s)
[ 2024-05-01 10:14:35,595 ] 91 luthien5921 - INFO - 行政文書任意開示申出書 -  - 2015年08月19日 09時00分 - 2100年1月1日 00時00分 - 3895 - https://www.shinsei.e-aichi.jp/pref-aichi-police-u/offer/offerList_detailTop?tempSeq=3895 - False
[ 2024-05-01 10:14:35,596 ] 91 luthien5921 - INFO - 行政文書開示請求書 -  - 2015年08月19日 09時00分 - 2100年1月1日 00時00分 - 3896 - https://www.shinsei.e-aichi.jp/pref-aichi-police-u/offer/offerList_detailTop?tempSeq=3896 - False
[ 2024-05-01 10:14:35,597 ] 91 main_controller - INFO - Found 2 form(s)
[ 2024-05-01 10:14:35,597 ] 91 main_controller - INFO - Customers: [{'first_name': 'The Hieu', 'last_name': 'Truong', 'date_birth': '19990506', 'nation': 'ベトナム', 'country': 'ベトナム', 'phone_number': '080-1583-0891', 'gender': 'M'}, {'first_name': 'Xuan Truong', 'last_name': 'Do', 'date_birth': '19921211', 'nation': 'ベトナム', 'country': 'ベトナム'}, {'first_name': 'Anh Ngoc', 'last_name': 'Chia', 'date_birth': '19961208', 'nation': 'ベトナム', 'country': 'ベトナム'}, {'first_name': 'Van Sang', 'last_name': 'Phan', 'date_birth': '19970228', 'nation': 'ベトナム', 'country': 'ベトナム'}, {'first_name': 'Hoai Phong', 'last_name': 'Vo', 'date_birth': '19981101', 'nation': 'ベトナム', 'country': 'ベトナム'}, {'first_name': 'Tuan Anh', 'last_name': 'Huynh', 'date_birth': '19950106', 'nation': 'ベトナム', 'country': 'ベトナム'}, {'first_name': 'Van Han', 'last_name': 'Nguyen', 'date_birth': '19910525', 'nation': 'ベトナム', 'country': 'ベトナム'}]
[ 2024-05-01 10:14:35,599 ] 91 main_controller - INFO - Customer info index 0: {'first_name': 'The Hieu', 'last_name': 'Truong', 'date_birth': '19990506', 'nation': 'ベトナム', 'country': 'ベトナム', 'phone_number': '080-1583-0891', 'gender': 'F'}
[ 2024-05-01 10:14:35,600 ] 91 main_controller - INFO - Customer index 0: Truong The Hieu, F, 19990506, 080-1583-0891, ベトナム
[ 2024-05-01 10:14:35,601 ] 91 luthien5921 - INFO - Accessing form, template sequence: 3895
[ 2024-05-01 10:14:35,601 ] 91 luthien5921 - INFO - Form URL: https://www.shinsei.e-aichi.jp/pref-aichi-police-u/offer/offerList_detailTop?tempSeq=3895
[ 2024-05-01 10:14:41,031 ] 91 luthien5921+shinsei2 - INFO - Accessed form successfully
[ 2024-05-01 10:14:41,052 ] 79 luthien5921+shinsei2 - ERROR - Failed to get form message, maybe form is available
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\filler.py", line 122, in get_form_message
    message = browser.find_element(By.XPATH, "/html/body/main/div/form/div[3]/strong/ul/li/span/strong").text
  File "C:\Users\USER\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "C:\Users\USER\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "C:\Users\USER\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "C:\Users\USER\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/main/div/form/div[3]/strong/ul/li/span/strong"}
  (Session info: chrome=124.0.6367.63); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF7B2A7F922+3045234]
	(No symbol) [0x00007FF7B2723652]
	(No symbol) [0x00007FF7B25E7564]
	(No symbol) [0x00007FF7B2635F9D]
	(No symbol) [0x00007FF7B263606C]
	(No symbol) [0x00007FF7B267BDD7]
	(No symbol) [0x00007FF7B2659DEF]
	(No symbol) [0x00007FF7B26791C4]
	(No symbol) [0x00007FF7B2659B53]
	(No symbol) [0x00007FF7B262813C]
	(No symbol) [0x00007FF7B2628D71]
	GetHandleVerifier [0x00007FF7B2AAE89D+3237613]
	GetHandleVerifier [0x00007FF7B2AF0BA7+3508727]
	GetHandleVerifier [0x00007FF7B2AE976F+3478975]
	GetHandleVerifier [0x00007FF7B285F1C6+815126]
	(No symbol) [0x00007FF7B272ED0F]
	(No symbol) [0x00007FF7B2729854]
	(No symbol) [0x00007FF7B27299E2]
	(No symbol) [0x00007FF7B271A574]
	BaseThreadInitThunk [0x00007FFFFB45257D+29]
	RtlUserThreadStart [0x00007FFFFCEEAA48+40]

[ 2024-05-01 10:14:41,060 ] 91 luthien5921+shinsei2 - INFO - Form is available, template sequence: 3896
[ 2024-05-01 10:14:41,060 ] 91 luthien5921+shinsei2 - INFO - Filling form, template sequence: 3896
[ 2024-05-01 10:14:41,297 ] 91 luthien5921 - INFO - Accessed form successfully
[ 2024-05-01 10:14:41,319 ] 79 luthien5921 - ERROR - Failed to get form message, maybe form is available
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\filler.py", line 122, in get_form_message
    message = browser.find_element(By.XPATH, "/html/body/main/div/form/div[3]/strong/ul/li/span/strong").text
  File "C:\Users\USER\anaconda3\envs\aichi\lib\site-packages\seleniumbase\core\sb_driver.py", line 20, in find_element
    return self.driver.default_find_element(by=by, value=value)
  File "C:\Users\USER\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "C:\Users\USER\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "C:\Users\USER\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"/html/body/main/div/form/div[3]/strong/ul/li/span/strong"}
  (Session info: chrome=124.0.6367.63); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Stacktrace:
	GetHandleVerifier [0x00007FF7B2A7F922+3045234]
	(No symbol) [0x00007FF7B2723652]
	(No symbol) [0x00007FF7B25E7564]
	(No symbol) [0x00007FF7B2635F9D]
	(No symbol) [0x00007FF7B263606C]
	(No symbol) [0x00007FF7B267BDD7]
	(No symbol) [0x00007FF7B2659DEF]
	(No symbol) [0x00007FF7B26791C4]
	(No symbol) [0x00007FF7B2659B53]
	(No symbol) [0x00007FF7B262813C]
	(No symbol) [0x00007FF7B2628D71]
	GetHandleVerifier [0x00007FF7B2AAE89D+3237613]
	GetHandleVerifier [0x00007FF7B2AF0BA7+3508727]
	GetHandleVerifier [0x00007FF7B2AE976F+3478975]
	GetHandleVerifier [0x00007FF7B285F1C6+815126]
	(No symbol) [0x00007FF7B272ED0F]
	(No symbol) [0x00007FF7B2729854]
	(No symbol) [0x00007FF7B27299E2]
	(No symbol) [0x00007FF7B271A574]
	BaseThreadInitThunk [0x00007FFFFB45257D+29]
	RtlUserThreadStart [0x00007FFFFCEEAA48+40]

[ 2024-05-01 10:14:41,320 ] 91 luthien5921 - INFO - Form is available, template sequence: 3895
[ 2024-05-01 10:14:41,321 ] 91 luthien5921 - INFO - Filling form, template sequence: 3895
[ 2024-05-01 10:14:42,822 ] 91 luthien5921+shinsei2 - INFO - Clicked continue button
[ 2024-05-01 10:14:42,822 ] 91 luthien5921+shinsei2 - INFO - Save page source to onlyday.html
[ 2024-05-01 10:14:42,837 ] 79 main_controller - ERROR - Caught exception: 'str' object is not callable
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 97, in subprocess
    filler.fill(browser, form_info, customer)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\filler.py", line 70, in fill
    pg_source = browser.page_source()
TypeError: 'str' object is not callable
[ 2024-05-01 10:14:43,245 ] 91 luthien5921 - INFO - Clicked continue button
[ 2024-05-01 10:14:43,246 ] 91 luthien5921 - INFO - Save page source to onlyday.html
[ 2024-05-01 10:14:43,261 ] 79 main_controller - ERROR - Caught exception: 'str' object is not callable
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 97, in subprocess
    filler.fill(browser, form_info, customer)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\filler.py", line 70, in fill
    pg_source = browser.page_source()
TypeError: 'str' object is not callable
[ 2024-05-01 10:14:45,259 ] 91 main_controller - INFO - Subprocess 2 started with trial number 1
[ 2024-05-01 10:14:45,737 ] 91 main_controller - INFO - Browser opened
[ 2024-05-01 10:14:46,094 ] 91 main_controller - INFO - Subprocess 0 started with trial number 1
[ 2024-05-01 10:14:48,328 ] 91 main_controller - INFO - Browser opened
[ 2024-05-01 10:14:50,580 ] 91 main_controller - INFO - Browser opened
